client/my-sites/domain-tip/index.jsx

```diff
@@ -46,10 +46,21 @@ const DomainTip = React.createClass( {
 		return true;
 	},
 
+	domainUpgradeNudge() {
+		return (
+			<UpgradeNudge
+					title={ this.translate( 'Get a free Custom Domain' ) }
+					message={ this.translate( 'Custom domains are free when you upgrade to a Premium or Business plan.' ) }
+					feature={ FEATURE_CUSTOM_DOMAIN }
+					event="stats_insights_domain"
```

Comment:
👍 Good catch. I've added an event prop in 9e0b00d. Are we ok with slightly redundant event names? The tip itself will then have the event of `domain_tip_stats_insights_domain`